internal/mutators/podtemplatespec.go

```diff
@@ -0,0 +1,113 @@
+package mutators
+
+import (
+	"fmt"
+
+	"github.com/kanopy-platform/hedgetrimmer/internal/limitrange"
+	"github.com/kanopy-platform/hedgetrimmer/internal/quantity"
+	corev1 "k8s.io/api/core/v1"
+)
+
+type PodTemplateSpec struct{}
+
+func NewPtsMutator() PodTemplateSpec {
```

Comment:
nit: This constructor is not necessary.

Reply:
We ended up needing this now because we'd like to make the `defaultMemoryLimitRequestRatio` configurable via the constructor.